pfb-analysis/scripts/entrypoint.sh

```diff
@@ -57,5 +57,5 @@ EOF
 bash
 
 # shutdown postgres
-kill $POSTGRES_PROC
+su postgres -c "/usr/lib/postgresql/9.6/bin/pg_ctl stop"
```

Comment:
Yeah, I was getting analysis containers not exiting, and when I opened a shell and ran `ps` there were postgres processes that seemed to just be hanging out.  Maybe that's not happening anymore for other reasons, but this seems no worse than the `kill` in any case.